src/styles/app/15-all-docs.css

```
@@ -115,13 +122,90 @@
+.allDocsCardPath {
+	min-width: 0;
+	display: inline-flex;
+	align-items: center;
+	gap: 4px;
+	overflow: hidden;
+	white-space: nowrap;
+}
+
+.allDocsCardPathIcon {
+	flex: 0 0 auto;
+	color: currentColor;
```

Stylelint (17.11.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)